RFS/src/main/java/com/rfs/common/SourceRepoAccessor.java

```diff
@@ -0,0 +1,47 @@
+package com.rfs.common;
```

Comment:
Please run formatter of your choice of this file any for any other changes to make whitespace and indenting consistent

Reply:
Reformatted, thanks.